core/error/error_macros.h

```diff
@@ -403,6 +403,13 @@ void _err_flush_stdout();
 	} else                                                                                             \
 		((void)0)
 
+#define ERR_FAIL_COND_STDOUT(m_cond)                                                                          \
```

Comment:
```suggestion
/**
 * Ensures `m_cond` is false.
 * If `m_cond` is true, the current function returns, and the error is printed directly
 * to stderr, bypassing the MessageQueue. This is useful to use in core templates
 * used by MessageQueue itself, to prevent infinite error loops.
 */
#define ERR_FAIL_COND_STDOUT(m_cond)                                                                          \
```